buildmongo/README.md

```diff
@@ -42,6 +44,8 @@ Based on: - [Setting-up BigQuery](https://www.ncbi.nlm.nih.gov/sra/docs/sra-bigq
 
 3. In big query console, under 'sraproject' create a dataset named 'mastiffdata'
 
-## CREATE ACCESS IN BQTOMONGO.py:
+#### Create access in bqmongo.py
```

Comment:
```suggestion
#### Create access in buildmongo/bqmongo.py
```

since you're reading it in from the toplevel directory now, would be good to point folks to where the file is